src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -388,6 +391,14 @@ private URL endpoint(String method) throws MalformedURLException {
         return new URL(getUrl(), "RealJenkinsRule/" + method + "?token=" + token);
     }
 
+    /**
+     * Obtains the Jenkins home directory.
+     * Normally it will suffice to use {@link LocalData} to populate files.
+     */
+    public File getHome() {
```

Comment:
Unused?

Reply:
Currently. I did start to use it in https://github.com/jenkinsci/workflow-cps-plugin/pull/489 to mount a subdir into an agent container, though this wound up not working due to https://github.com/testcontainers/testcontainers-java/discussions/4462. At any rate, it seems useful to be able to read or write files in the home dir from the test JVM.